### PR TITLE
Added dolang.py

### PR DESCRIPTION
**Pull request by albop**
Checklist
- [x ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [x ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x ] If static libraries are linked in, the license of the static library is packaged.
- [x ] Build number is 0
- [x ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/dolang/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: de602c48c7b4b6809b8b450cb4a2e3b45408ba372e656f182b93d4508467f2e4
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```
Comment (synapticarbors) [inline recipes/dolang/meta.yaml:18]: Are the other packages besides `pip`, `python` and `poetry` needed to build the package?

File: recipes/dolang/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - dolang
+    - dolang.tests
+
+about:
+  home: https://github.com/EconForge/dolang.py
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: COPYING
```
Comment (synapticarbors) [inline recipes/dolang/meta.yaml:42]: about the `license_file` like, put a comment containing the link the following issue as a reminder to check whether the license file is include in the source distribution for future releases (and so the manually included copy can be removed):

https://github.com/EconForge/dolang.py/issues/3

Comment (albop) [reply recipes/dolang/meta.yaml:41]: I did that and the issue is solved in upstream dolang repository. Turns out poetry includes COPYING automatically, not LICENSE. In the future, poetry will be able to include either of both but in the meantime I just moved copying to COPYING.

File: recipes/dolang/meta.yaml (hunk 1 above)
Comment (albop) [reply recipes/dolang/meta.yaml:18]: No, you're right, I removed the other ones.

**Pull request merged**